once_test.py

```diff
@@ -611,6 +664,41 @@ def run(self) -> int:
             a.counter.ready.set()
             self.assertEqual(a_job.result(timeout=5), 1)
 
+    def test_once_per_instance_parallel(self):
+        class _CallOnceClass(Counter):
+            @once.once_per_instance
+            def once_fn(self):
+                return self.get_incremented()
+
+        once_objs = [_CallOnceClass(), _CallOnceClass(), _CallOnceClass(), _CallOnceClass()]
+
+        @execute_with_barrier(n_workers=_N_WORKERS)
```

Comment:
Same here, this barrier placement isn't testing that they run in parallel (code could be changed to run instances in serial and the test would still pass).

If you make one instance per _N_WORKERS and put the barrier on the onced function, then it should test parallelness

Reply:
added a comment

Reply:
This test is missing a barrier inside the onced function that tests for parallel execution (also I don't see the comment). We should have a barrier with size 4 that tests for parallel execution

Reply:
ok actually resolved